cropper/src/main/java/com/canhub/cropper/CropImageOptions.kt

```diff
@@ -38,522 +38,312 @@ open class CropImageOptions : Parcelable {
      * Default value: true
      */
     @JvmField
-    var imageSourceIncludeCamera: Boolean
+    val imageSourceIncludeCamera: Boolean = true,
 
     /** The shape of the cropping window.  */
     @JvmField
-    var cropShape: CropShape
+    val cropShape: CropShape = CropShape.RECTANGLE,
     /**
      * The shape of cropper corners
      */
     @JvmField
-    var cornerShape: CropImageView.CropCornerShape
+    val cornerShape: CropImageView.CropCornerShape = CropImageView.CropCornerShape.RECTANGLE,
     /**
      * The radius of the circular crop corner
      */
     @JvmField
-    var cropCornerRadius: Float
+    val cropCornerRadius: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 10f, Resources.getSystem().displayMetrics),
 
     /**
      * An edge of the crop window will snap to the corresponding edge of a specified bounding box when
      * the crop window edge is less than or equal to this distance (in pixels) away from the bounding
      * box edge. (in pixels)
      */
     @JvmField
-    var snapRadius: Float
+    val snapRadius: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 3f, Resources.getSystem().displayMetrics),
 
     /**
      * The radius of the touchable area around the handle. (in pixels)<br></br>
      * We are basing this value off of the recommended 48dp Rhythm.<br></br>
      * See: http://developer.android.com/design/style/metrics-grids.html#48dp-rhythm
      */
     @JvmField
-    var touchRadius: Float
+    val touchRadius: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 24f, Resources.getSystem().displayMetrics),
```

Comment:
This should never be used as it might be different from the current window.
Alternatively I'd just rename this to `touchRadiusDp`, set 24 as the default value and do the conversion at a later stage.

Reply:
agreed, but would suggest another PR to do the changes, so we keep it self contained